bindings/nodejs/lib/types/models/storage-score.ts

```diff
@@ -10,25 +10,25 @@ export interface StorageScoreParameters {
     /**
      *  Defines the number of IOTA tokens required per unit of storage score.
      */
-    storageCost: u64;
+    storageCost: NumericString;
```

Comment:
Is there no way having u64 behave as NumericString? What was the point of introducing it if we're removing it more and more?

Reply:
It only works in classes, since we there can convert it during construction of the type, but the interfaces are just type hints, so no conversion possible (unless we change them to classes)

Reply:
I hate this

Reply:
Why not keep u64 anyway if conversion doesn't work?

Reply:
![image](https://github.com/iotaledger/iota-sdk/assets/46689931/67d658a7-f7b7-4c6b-bd87-49b0fea47924)

Reply:
> Why not keep u64 anyway if conversion doesn't work?

Because the returned type is a string, so u64 would be wrong

Reply:
Ok